x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/TransportPutFollowAction.java

```diff
@@ -135,6 +140,12 @@ private void createFollowerIndex(
             return;
         }
 
+        if (leaderIndexMetadata.isSystem()) {
+            deprecationLogger.deprecate(DeprecationCategory.INDICES,
```

Comment:
This deprecation logging will fire on any explicit system index follow, whereas the breaking change in #72815 only skips auto-following system indices.

I would prefer to have the deprecation logging for auto-following only.

Reply:
Yes, that's something we need to discuss. Currently there's no way to know if a followed index request come from AutoFollow or from an explicit user request. I was wondering if we should add some extra metadata in order to distinguish the following origin, in that case existing followed system indices won't show up in the deprecation API. wdyt @henningandersen?